tests/qwatch_test.go

```diff
@@ -97,11 +98,11 @@ func TestQWATCHWithSDK(t *testing.T) {
 	subscribers := []*redis.Client{getLocalSdk(), getLocalSdk(), getLocalSdk()}
 
 	// Cleanup Store for next tests
-	defer func() {
-		for _, tc := range qWatchTestCases {
-			publisher.Del(context.Background(), fmt.Sprintf("match:100:user:%d", tc.userID))
-		}
+	for _, tc := range qWatchTestCases {
+		publisher.Del(context.Background(), fmt.Sprintf("match:100:user:%d", tc.userID))
+	}
```

Comment:
We should move this to the end of the function so that when this test exits it cleans up for the other tests in the suite (instead of other tests having to worry about cleaning up the store before they execute.